Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
Just remember, that could take a significantly long time.  Also, in the case of info, you have to pass a switch to check remotely.  @rhatdan are you suggesting that if the image doesnt exist locally, we ignore that switch and look remotely as well?